script.js

```diff
@@ -1,16 +1,105 @@
+    listedPlanets = "https://handlers.education.launchcode.org/static/planets.json";
 
-window.addEventListener("load", function() {
-
-   let listedPlanets;
-   // Set listedPlanetsResponse equal to the value returned by calling myFetch()
-   let listedPlanetsResponse;
-   listedPlanetsResponse.then(function (result) {
-       listedPlanets = result;
-       console.log(listedPlanets);
-   }).then(function () {
-       console.log(listedPlanets);
-       // Below this comment call the appropriate helper functions to pick a planet fom the list of planets and add that information to your destination.
-   })
-   
-});
\ No newline at end of file
+    // Fetching Planetary Data
+    this.fetch(listedPlanets).then(function(result) {
+        //listedPlanets=results;
+        result.json().then(function(json) {
+            let listedPlanetsResponse = document.getElementById("missionTarget");
+            //console.log(listedPlanets);
+            let missionDestination = Math.floor(Math.random()*json.length -1);
+           listedPlanetsResponse.innerHTML = `
+           <h1>Mission Destination</h1>
+           <ol>
+              <li>Name: ${json[missionDestination].name}</li>
+              <li>Diameter: ${json[missionDestination].diameter}</li>
+              <li>Star: ${json[missionDestination].star}</li>
+              <li>Distance from Earth: ${json[missionDestination].distance}</li>
+              <li>Number of Moons: ${json[missionDestination].moons}</li>
+           </ol>
+           <img src="${json[missionDestination].image}"> `;
+        });
+     });
+
+
+//Submitting pilot, co-pilot, fuel level, & cargo mass
+let form = document.querySelector("form");
+form.addEventListener("submit", function(event){
+    let pilotName = document.querySelector("input[name=pilotName]");
+    let copilotName = document.querySelector("input[name=copilotName]");
+    let fuelLevel = document.querySelector("input[name=fuelLevel]");
+    let cargoMass = document.querySelector("input[name=cargoMass]");
+
+    let faultyItems = document.getElementById('faultyItems');
+    let fuelStatus = document.getElementById('fuelStatus');
+    let launchStatus = document.getElementById('launchStatus');
+    let pilotStatus = document.getElementById('pilotStatus');
+    let copilotStatus = document.getElementById('copilotStatus');
+
+
+
+    // Validate user enters something for every field
+
+    if (pilotName.value.trim() === ""|| copilotName.value.trim() === "" || fuelLevel.value.trim() === "" || cargoMass.value.trim() === "") {
+        alert("All fields are required!");
+        event.preventDefault();
+    }
+
+    //Validate user enters text for names and numbers for fuel and cargo levels
+
+    if (isNaN (pilotName.value)|| isNaN(copilotName.value)) {
+        pilotStatus.innerHTML =`Pilot ${pilotName.value} is ready`;
+        copilotStatus.innerHTML =`Pilot ${copilotName.value} is ready`;
```

Comment:
```suggestion
        pilotStatus.innerHTML =`Pilot ${pilot} is ready`;
        copilotStatus.innerHTML =`Pilot ${copilot} is ready`;
```

Once this code is move into your `formSubmission()` function in `scriptHelper.js`, you will need to change the name to be `pilot` and `copilot` to use the parameter names declared in the function's parenthesis.